### ISSUE TITLE
[bug?] - Shields

### ISSUE THREAD

**Issue by MrAlinoe** [CreativeMD/EnhancedVisuals#37]
Minor bug, with vanilla shields, bloodsplatters displayed when the shield block a arrow...

**Comment by CreativeMD** [comment]
This issue will be fixed once the rework of EnhancedVisuals is done.

**Comment by CreativeMD** [comment]
This should be fixed in v1.4.0, if not open a new issue.
